### ISSUE TITLE
Replace introductory RapidFTR text on homepage of RapidREG

### ISSUE THREAD

**Issue by tomclement** [rapidftr/tracker#290]

**Comment by nellyk** [comment]
@tomclement hi We would like to work on this issue, which specific text would you like for it to be changed , can you kindly provide us with a screenshot?

**Comment by tomclement** [comment]
Hi @nellyk - Thanks for taking interest in resolving this issue. However I just checked and this was actually resolved by [this commit](https://github.com/rapidftr/RapidFTR/commit/4718d83586937d0dda5b4e544a22c141e97775fd) from @austiine04 - so there is no further work to do in this regard. I'm therefore closing this.

For reference, the text that was changed is shown in the screenshot below. (The text changes based on whether the running app is an instance of RapidFTR or RapidREG)

<img width="989" alt="welcome-screen" src="https://cloud.githubusercontent.com/assets/5906754/9032459/8bbe0adc-39c7-11e5-8ece-eebc845039e2.png">
